config/cmake/ConfigureChecks.cmake

```diff
@@ -547,9 +547,9 @@ set (${HDF_PREFIX}_DEFAULT_VFD H5FD_SEC2)
 
 if (NOT DEFINED "${HDF_PREFIX}_DEFAULT_PLUGINDIR")
   if (WINDOWS)
-    set (${HDF_PREFIX}_DEFAULT_PLUGINDIR "%ALLUSERSPROFILE%\\\\hdf5\\\\lib\\\\plugin")
+    set (${HDF_PREFIX}_DEFAULT_PLUGINDIR "${CMAKE_INSTALL_PREFIX}\\\\lib\\\\plugin;%ALLUSERSPROFILE%\\\\hdf5\\\\lib\\\\plugin")
   else ()
-    set (${HDF_PREFIX}_DEFAULT_PLUGINDIR "/usr/local/hdf5/lib/plugin")
+    set (${HDF_PREFIX}_DEFAULT_PLUGINDIR "${CMAKE_INSTALL_PREFIX}/lib/plugin:/usr/local/hdf5/lib/plugin")
```

Comment:
Won't this have issues with lib64?

Reply:
Likely not, because the plugins and libs are stored under the default install location. The actual setting can be changed by variables. Of greater concern, is the established path manipulations that happen in configure phase. Those assumptions and settings probably need reviewed and updated.

Reply:
> Likely not, because the plugins and libs are stored under the default install location.

But the default install location could just as likely be under lib64 couldn't it?

> The actual setting can be changed by variables. Of greater concern, is the established path manipulations that happen in configure phase. Those assumptions and settings probably need reviewed and updated.

Agreed.